modules/oadp-self-signed-certificate.adoc

```diff
@@ -38,4 +38,4 @@ spec:
 ...
 ----
 <1> Specify the Base46-encoded CA certificate string.
-<2> Must be `false` to disable SSL/TLS security.
+<2> Enter `true` to disable TLS/SSL security. Enter `false` to enable TLS/SSL security.
```

Comment:
<2>  The `insecureSkipTLSVerify` config can be either `"true"` or `"false"`. If set to true, TLS/SSL security is disabled. If set to false, TLS/SSL security is enabled.